docs/ramalama-serve.1.md

```diff
@@ -39,7 +39,7 @@ Generate specified configuration format for running the AI Model as a service
 show this help message and exit
 
 #### **--host**="0.0.0.0"
-ip address to listen
```

Comment:
Doesn't block merge, but IP address to listen made more sense to me, we will have at least 2 other servers soon llama-cpp-python and vllm